modules/local/report.nf

```diff
@@ -130,14 +130,13 @@ def generate_coverage_data(sample_data, bp_field, species){
                 && species[species_data_pos].fixed_genome_size != null){
 
                 def length = species[species_data_pos].fixed_genome_size.toLong()
-                def cov = base_pairs / q_length
+                def cov = base_pairs / length
```

Comment:
Just checking to make sure I understand the difference between q_length and length:

- is `q_length` a dynamic value specific to each sample entry in `sample_data`?
- is `length` a fixed value of the species genome size, as in a standard reference for each species.?

Reply:
This is correct! however, I think I have something mixed up. Just reading through the code again to make sure I am reporting the correct thing.

Reply:
`q_length` or query length here is the genome length generated by quast in this case or whatever path is pointed to in the `QCReportFields` of the `nextflow.config` while the length field is the length specified in the `QCReport` section for a specific organism.

Reply:
Fantastic! Thanks for looking into this!